src/users/users.ts

```diff
@@ -217,4 +219,18 @@ export class Users {
 
     return deserializeUser(data);
   }
+
+  async updateUserPassword({
+    userId,
+    password,
+  }: UpdateUserPasswordOptions): Promise<User> {
+    const { data } = await this.workos.put<
+      UserResponse,
+      SerializedUpdateUserPasswordOptions
+    >(`/users/${userId}/password`, {
```

Comment:
Should we have a dedicated serialization function like we do here? https://github.com/workos/workos-node/blob/c99d90819c6241d3612f18f42aaf62195f877a58/src/organizations/organizations.ts#L40